src/docs/product/session-replay/index.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: "Session Replay"
+sidebar_order: 3
+redirect_from:
+  - /session-replay/
+description: "Session Replay. TODO"
+---
```

Comment:
New item in the sidebar of the Product section:
<img width="294" alt="SCR-20221205-nph" src="https://user-images.githubusercontent.com/187460/205780519-31d8d19f-d76f-444a-8d30-aa1044a50366.png">